sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/MetricCell.java

```diff
@@ -28,7 +29,17 @@
  * @param <DataT> The type of metric data stored (and extracted) from this cell.
  */
 @Experimental(Kind.METRICS)
-public interface MetricCell<UserT extends Metric, DataT> {
+public interface MetricCell<UserT extends Metric, DataT> extends Serializable {
+
+  /**
+   * Update value of this cell.
+   */
+  void update(DataT data);
```

Comment:
Does this do a set or a merge with the given data? If it does a set, then the method below should be called `merge` or something else to distinguish the behaviors. Either way, doc should indicate.